Documentation/ContentObjects/Case/Index.rst

```diff
@@ -26,32 +26,70 @@ be used as it has a special meaning: If the value of the "key"
 property is *not* found in the array of cObjects, then the cObject
 from the "default" property will be used.
 
+.. contents::
+   :local:
 
 .. ### BEGIN~OF~TABLE ###
 
+Properties
+==========
+
+
+1,2,3,4...
```

Comment:
As I understand, that can also be strings as it is the value of the selected key. I would avoid 1,2,3,4... as it is near the default usage of a COA, for example (first I thought about that). I would leave it to the previous "(array of cObjects)"

Reply:
true, I mixed this up with the COA